spec/policies/user_policy_spec.rb

```diff
@@ -42,18 +42,6 @@
         expect(policy_scope.pluck(:id).sort).to eq([other_member.id, current_user.id].sort)
       end
     end
-
-    context "for a user acting on himself" do
```

Comment:
Only new and create should be removed from these tests, no? 

Also, you could test the opposite, should not, for these.

Last, you could start changing this should to is_expected...